bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/ThreadUtil.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase.util;
+
+import com.google.api.core.InternalApi;
+import com.google.cloud.PlatformInformation;
+import com.google.common.util.concurrent.MoreExecutors;
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import java.util.concurrent.ThreadFactory;
+
+/**
+ * Creates a {@link ThreadFactory} that's safe to use in App Engine.
+ *
+ * <p>This class copies code that originates in {@link
+ * io.grpc.internal.GrpcUtil#getThreadFactory(String, boolean)}.
+ *
```

Comment:
Are you sure that this note should be here?   (I don't know, just asking)

Reply:
Yes, It still holds true for our utility. So it's best to keep track of it.